packages/sanity/src/core/studio/screens/NotAuthenticatedScreen.tsx

```diff
@@ -51,8 +51,8 @@ export function NotAuthenticatedScreen() {
       >
         <Stack space={4}>
           <Text>
-            You are not authorized to access this studio. Maybe you could ask someone to invite you
-            to collaborate on this project?
+            You are not authorized to access this studio. Please contact someone with access to
```

Comment:
COULD: what do you think to localizing this and the RequestAccessScreen too in `packages/sanity/src/core/i18n/bundles/studio.ts`?

Reply:
We're hoping to get the functionality out in the next release and add more to this PR (will be adding testing though), I'll make a ticket to follow up with adding translations to these two screens